ui-participant/src/hub/AlertComponent.tsx

```diff
@@ -0,0 +1,24 @@
+import { Enrollee } from '../api/api'
+import React from 'react'
+
+/**
+ *
+ */
+export default function AlertComponent({ enrollee } :{enrollee:Enrollee}) {
+  function getNumberOfTasks(enrollee: Enrollee) {
+    let waitingTasks = 0
+    enrollee.participantTasks.forEach(participantTask => {
+      if (participantTask.status != 'COMPLETE') {
+        waitingTasks++
+      }
+    })
+    return waitingTasks
+  }
+  return (<span>
+    <span
+      className={`rounded-circle alert-circle
+    ${getNumberOfTasks(enrollee) == 0 ? 'bg-secondary-subtle' : 'bg-danger text-white'}`}>
+      {getNumberOfTasks(enrollee)}
+    </span>
+  </span>)
+}
```

Comment:
I think this can be simplified a bit so you don't need the `getNumberOfTasks` function, i.e.:

```suggestion
export default function AlertComponent({ enrollee } :{enrollee:Enrollee}) {
  const numTasks = enrollee.participantTasks.filter(participantTask => participantTask.status != 'COMPLETE').length

  return (<span>
    <span className={`rounded-circle alert-circle
      ${numTasks == 0 ? 'bg-secondary-subtle' : 'bg-danger text-white'}`
    }>
      {numTasks}
    </span>
  </span>)
}
```